### DISCUSSION TITLE
Regularly run document_renamer (or run after tag / medata updates that would impact file layout)

### DISCUSSION THREAD

**Discussion by lenucksi** [jonaswinkler/paperless-ng#805] [Feature Requests]
This related to the #803 feature request.

I'm using a on disk structure dependent on document type, date and correspondent. Sometimes I have to change the correspondent, e.g. because it has been wrongly assigned by classifiers or something else changed, This happens on a individual document or small batch level.
Running the `document_renamer` management command would be an optio here, even running it on a cron. However I think this might be a bit excessive. Would it be an option to do the individual moving directly after editing a document? Or to have the renamer run as a regular task on normal operation?

Having a (filtered) log of such operations might also be great.

**Comment by jonaswinkler** [comment]
Paperless already moves files after you change the associated documents in the web UI. Calling the document_renamer manually should only be required after changing PAPERLESS_FILENAME_FORMAT. I am aware of #750. If you encounter any other cases where paperless does not update filenames, that's a bug.

**Reply by lenucksi** [reply]
Thanks, I'll look into the bug and how see what I can reproduce on the instance here.
